src/System.Memory/tests/SequenceReader/CopyTo.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using Xunit;
+
+namespace System.Memory.Tests.SequenceReader
+{
+    public class CopyTo
+    {
+        [Fact]
+        public void TryCopyTo_Empty()
+        {
+            var reader = new SequenceReader<char>(ReadOnlySequence<char>.Empty);
+
+            // Nothing to nothing is always possible
+            Assert.True(reader.TryCopyTo(Span<char>.Empty));
+
+            // Nothing to something doesn't work
+            Assert.False(reader.TryCopyTo(new char[1]));
```

Comment:
Hmm. Maybe this should return true. The destination was large enough to fit the empty span.

Isn't false supposed to mean the destination was too small?

Reply:
cc @davidfowl, thoughts?